phys/module_diag_trad_fields.F

```diff
@@ -112,6 +121,20 @@ SUBROUTINE trad_fields ( u,v,w,t,qv,zp,zb,pp,pb,p,pw,            &
                   END IF
                END IF
 
+               !  Potential Temperature  
+
+               IF ( use_theta_m .EQ. 1 ) THEN
+                  temperature(i,k,j) =  ( t(i,k,j) + t0 )  / ( 1. + (R_v/R_d) * qv(i,k,j) )
+               ELSE
+                  temperature(i,k,j) =   t(i,k,j) + t0  
+               END IF
+
+
+               !  Relative humidity
+
+                  ptot(i,k,j) = pb(i,k,j)+pp(i,k,j)
```

Comment:
@smileMchen 
Ming,
If this is the only usage of ptot, you could save a local 3d array and just pass in a scalar ptot. That means the declaration above for ptot gets changed to a scalar.

Reply:
@davegill 
Dave,
I agree. I have made changes as you suggested.